crowdstrike/table_crowdstrike_spotlight_vulnerability.go

```diff
@@ -0,0 +1,196 @@
+package crowdstrike
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/crowdstrike/gofalcon/falcon"
+	"github.com/crowdstrike/gofalcon/falcon/client"
+	"github.com/crowdstrike/gofalcon/falcon/client/spotlight_vulnerabilities"
+	"github.com/crowdstrike/gofalcon/falcon/models"
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+	"golang.org/x/time/rate"
+)
+
+func tableCrowdStrikeSpotlightVulnerability(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "crowdstrike_spotlight_vulnerability",
+		Description: "TODO.",
+		Get: &plugin.GetConfig{
+			Hydrate:    getCrowdStrikeSpotlightVulnerability,
+			KeyColumns: plugin.SingleColumn("id"),
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listCrowdStrikeSpotlightVulnerabilities,
+			KeyColumns: []*plugin.KeyColumn{
+				{
+					Name:      "aid",
+					Require:   plugin.Optional,
+					Operators: []string{"="},
+				},
+				{
+					Name:      "status",
+					Require:   plugin.Optional,
+					Operators: []string{"="},
+				},
+				{
+					Name:      "closed_timestamp",
+					Require:   plugin.Optional,
+					Operators: []string{">", ">=", "=", "<", "<="},
+				},
+				{
+					Name:      "created_timestamp",
+					Require:   plugin.Optional,
+					Operators: []string{">", ">=", "=", "<", "<="},
+				},
+				{
+					Name:      "updated_timestamp",
+					Require:   plugin.Optional,
+					Operators: []string{">", ">=", "=", "<", "<="},
+				},
+			},
+		},
+		Columns: []*plugin.Column{
+			{Name: "cid", Description: "The Customer ID.", Type: proto.ColumnType_STRING},
+			{Name: "aid", Description: "TODO", Type: proto.ColumnType_STRING},
+			{Name: "app", Description: "TODO", Type: proto.ColumnType_JSON},
+			{Name: "apps", Description: "TODO", Type: proto.ColumnType_JSON},
+			{Name: "closed_timestamp", Description: "TODO", Type: proto.ColumnType_TIMESTAMP},
+			{Name: "created_timestamp", Description: "TODO", Type: proto.ColumnType_TIMESTAMP},
+			{Name: "cve", Description: "TODO", Type: proto.ColumnType_JSON},
+			{Name: "host_info", Description: "TODO", Type: proto.ColumnType_JSON},
+			{Name: "id", Description: "TODO", Type: proto.ColumnType_STRING},
+			{Name: "remediation", Description: "TODO", Type: proto.ColumnType_JSON},
+			{Name: "status", Description: "TODO", Type: proto.ColumnType_STRING},
+			{Name: "updated_timestamp", Description: "TODO", Type: proto.ColumnType_TIMESTAMP},
+
+			// Steampipe standard columns
+			{Name: "title", Description: "Title of the resource.", Type: proto.ColumnType_STRING, Transform: transform.FromField("Cid")},
```

Comment:
Is this the best title for a spotlight vulnerability? The customer ID seems like related info, but wouldn't always tell me how to find the vulnerability (difficult to tell without the other column IDs though)

Reply:
This is probably not. I am changing it to ID, but event that is probably not a good choice